create-html-reports.sh

```diff
@@ -154,14 +158,19 @@ if [ "$ALL_REPORTS" != "" ]; then
    " >> $report
     for f in $ALL_REPORTS; do
         report_name=$(basename -s .txt $f)
+        # report_path should be of the form "testing/foo.html"
```

Comment:
I believe it would be of the form `tests/foo.html`

Reply:
updated